package.json

```diff
@@ -4,7 +4,11 @@
   "dependencies": {
     "autoprefixer": "^6.5.4",
     "babel-preset-es2015": "^6.22.0",
+    "bootstrap": "^3.3.7",
     "del": "^2.2.2",
+    "fitvids": "^2.0.0",
```

Comment:
Sorry, didn't catch this the first time.

Let's remove the `fitvids` package since we only need `fitvids.1.1.0`